src/main/kotlin/pl/put/smartgarden/infra/api/UserDeviceController.kt

```diff
@@ -16,129 +15,21 @@ import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.ResponseStatus
 import org.springframework.web.bind.annotation.RestController
-import pl.put.smartgarden.domain.device.dto.request.MeasureRequest
 import pl.put.smartgarden.domain.device.dto.response.MeasureResponse
 import pl.put.smartgarden.domain.device.dto.response.SensorResponse
-import pl.put.smartgarden.domain.user.UserService
-import pl.put.smartgarden.domain.user.dto.response.AreaResponse
-import pl.put.smartgarden.domain.user.dto.response.AreaSettingsResponse
+import pl.put.smartgarden.domain.user.UserDeviceService
 import pl.put.smartgarden.domain.user.dto.request.IrrigationLevelRequest
-import pl.put.smartgarden.domain.user.dto.request.LocationRequest
 import pl.put.smartgarden.domain.user.dto.request.NextIrrigationRequest
-import pl.put.smartgarden.domain.user.dto.request.UserChangeEmailRequest
-import pl.put.smartgarden.domain.user.dto.request.UserChangePasswordRequest
-import pl.put.smartgarden.domain.user.dto.request.UserChangeUsernameRequest
-import pl.put.smartgarden.domain.user.dto.request.UserResourceResponse
-import pl.put.smartgarden.domain.user.dto.request.UserSignInRequest
-import pl.put.smartgarden.domain.user.dto.request.UserSignInResponse
-import pl.put.smartgarden.domain.user.dto.request.UserSignUpRequest
+import pl.put.smartgarden.domain.user.dto.response.AreaResponse
+import pl.put.smartgarden.domain.user.dto.response.AreaSettingsResponse
 import java.time.Instant
-import javax.validation.Valid
 
-@Api(description = "Users api")
+@Api(description = "User device frontend API")
 @RestController
```

Comment:
Nie mam czasu szukac lepszego miejsca na ten komentarz, więc wrzucam tu xd. Ten ErrorHandler ja bym wywalil z package api i dał do infry po prostu, bo to nie jest api przeciez :P

Reply:
👍